package.json

```diff
@@ -47,16 +48,18 @@
     "postcss-cli": "11.0.1",
     "postcss-import": "16.1.1",
     "postcss-inline-svg": "6.0.0",
+    "postcss-loader": "8.2.0",
     "postcss-preset-env": "10.5.0",
     "prettier-package-json": "2.8.0",
     "prop-types": "15.8.1",
-    "react": "18.3.1",
-    "react-dom": "18.3.1",
-    "react-is": "18.3.1",
+    "react": "19.2.3",
+    "react-dom": "19.2.3",
+    "react-is": "19.2.3",
```

Comment:
I'm not sure we should be bumping anything to React v19 until the Garden dependencies open up the peer dependency range.